figures/dp1_fields_depths.tex

```diff
@@ -0,0 +1,54 @@
+\begin{figure}[htbp]
+    \centering
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_47_tuc_r}
+        \caption{47 Tuc}
+        \label{fig:img1}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_rubin_sv_38_7_r}
+        \caption{Rubin SV 38 7}
+        \label{fig:img2}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_fornax_dsph_r}
+        \caption{Fornax dSph}
+        \label{fig:img3}
+    \end{subfigure}
+    \vspace{1em}  % Vertical space between rows
+     \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_ecdfs_r}
+        \caption{ECDFS}
+        \label{fig:img4}
+    \end{subfigure}
+    \hfill
+    \vspace{1em}  
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_edfs_r}
+        \caption{EDFS}
+        \label{fig:img6}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_rubin_sv_95_-25_r}
+        \caption{Rubin SV 95 -25}
+        \label{fig:img7}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{comcam_psf_maglim_seagull_r}
+        \caption{Seagull}
+        \label{fig:img8}
+    \end{subfigure}
+     \caption{Cumulative imaging depth expressed in terms of the $S/N=5$ limiting magnitude for unresolved sources for seven ComCam Deep Drilling Fields.}
```

Comment:
Is it really accurate to describe all seven as "Deep Drilling" fields? I think this is the only place in the document where that's done.

Reply:
yeah - this is a mistake - thanks. I don't say this elsewhere (nor in the paper)